examples/checkout/create_payment_request.rb

```diff
@@ -14,8 +14,7 @@
 
 # if you don't want use the application config, can give your credentials object to payment request
```

Comment:
but since it's using the credential object, the text should say, that the user can use the application config, instead credential object if wanted.